swatch-contracts/src/main/resources/swatch_contracts_openapi.yaml

```
@@ -183,6 +183,32 @@ paths:
         -
           test: []
       operationId: createContract
+  /internal/offerings/{sku}/product_tags:
+      description: "Mapping sku to product tags."
+      parameters:
+        - name: sku
+          in: path
+          required: true
+          schema:
+            type: string
+      get:
+        summary: "Lookup product tags by sku"
+        operationId: getSkuProductTags
+        responses:
+          '200':
+            description: "The request to get product tags by sku ."
+            content:
+              application/json:
+                schema:
+                  $ref: '#/components/schemas/OfferingProductTags'
+                example:
+                  data:
+                    - Rho one
+                    - Rho two
+        tags:
+          - Offerings
+        security:
+         - test: []
```

nitpick: inconsistent indentation:
```suggestion
          - test: []
```